### PR TITLE
New first responder badge credential for medical technician

### PR DESCRIPTION
**Pull request by jameseaster**
#### _Resolves - adds new first responder badge credential for medical technician_

---

### What kind of change does this PR introduce?

- New VC example for first responder badge.

<br/>

### What is the current behavior?

- No current FR EMT badge credential.

<br/>

### What is the new behavior?

- FR EMT badge credential.

NOTE: This uses the examples context instead of the appropriate context, because VC v2 and renderMethod are not yet stable.

<br/>

### Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

<br/>

### How has this been tested?

- Locally rendered, issued, and verified.

<br/>

### Screenshots: n/a

### CODE REVIEW

File: credentials/fr-medical-technician/landscape.svg
Comment (BigBlueHat) [inline credentials/fr-medical-technician/landscape.svg:1]: @jameseaster guessing you'd want a white background on this, correct?

**Comment by jameseaster** [comment]
@BigBlueHat
Medical Technician SVG is looking great here! A few notes:

- I'm seeing we are using 3 different FR blue star logos: Without words, with words in a circle (the iconography, and with horizontal words on issuer's image
- SVG needs rounded boarders to mimic a card
- SVG needs a background color as it currently blends into the UI background

You also noted:

- font issues
- Utopia background

File: credentials/fr-medical-technician/landscape.svg
Comment (jameseaster) [reply credentials/fr-medical-technician/landscape.svg:1]: Just now seeing this! We could do white, transparent blue (same blue that is used in the SVG), or whatever else you see fit!

**Comment by jameseaster** [comment]
For reference:

> SVG with values
>
> svg

**Comment by BigBlueHat** [comment]
> * I'm seeing we are using 3 different FR blue star logos: Without words, with words in a circle (the iconography, and with horizontal words on issuer's image

Not quite sure how to resolve these... One is the icon `image.png` used on the VC Playground, so I could drop the words there (as I expect they'll be illegible). The other one standalone image is for the issuer...so not keen to change their colors...and not keen to match that blue either. Any preference here, @jameseaster?

**Comment by BigBlueHat** [comment]
> * SVG needs rounded boarders to mimic a card

Not sure we need to do this one. The ecosystem will certainly do both (at various points). I can if you want me to, but I think it'll depend on the designer of the SVG in the future and since these sort of simulate "random" credentials...maybe it's fine? 😁 Let me know if you really want that changed.

**Comment by BigBlueHat** [comment]
All other issues _should_ be addressed.

**Comment by jameseaster** [comment]
@BigBlueHat 
> One is the icon image.png used on the VC Playground, so I could drop the words there (as I expect they'll be illegible). 
Let's match these, maybe just override the png file with a png version of the logo you're using in the svg

**Review by BigBlueHat — APPROVED** [review]
These should be good to go now. Inlined fonts, compressed, light blue background.

**Comment by jameseaster** [comment]
Rendered locally and looks awesome! Thanks @BigBlueHat

**Pull request merged**